src/lib/instanceManager/createInstanceManager.js

```diff
@@ -30,36 +32,38 @@ module.exports = ({ turbine, window, runAlloy, orgId }) => {
       // provides a backdoor to perform certain operations
       // synchronously, because Reactor requires that data
       // elements be resolved synchronously for now.
-      reactorRegisterGetEcid(getEcid) {
-        accessor.getEcid = getEcid;
-      },
```

Comment:
Notice I removed the getEcid stuff until we figure out how we're actually going to handle ECIDs (and it sounds like the command name will be changing too).